api/views/share.py

```diff
@@ -127,3 +130,11 @@ def get(self, request, *args, **kwargs):
                 return HttpSmartResponsePermanentRedirect(url)
             return HttpSmartResponseRedirect(url)
         return http.HttpResponseGone()
+
+
+class ModelTypesView(views.APIView):
+    def get(self, request, *args, **kwargs):
+        yaml_file = os.path.abspath('../SHARE/share/models/creative.yaml')
```

Comment:
This path will not work on all machines. Use the module level variable `__file__` to build the path.